Competition2018/src/main/java/competition/subsystems/climberdeploy/commands/ExtendClimberArmCommand.java

```diff
@@ -26,10 +26,9 @@ public void execute() {
 		deploy.extendClimberArm();
 		if (deploy.hitBarHeight()) {
```

Comment:
let's slide this logic around:

```
public boolean isFinished() {
    return deploy.hitBarHeight();
}

public void end() {
    deploy.stopClimberArm();
}

Reply:
end is like the opposite of `initialize()` - it's called once as the command ends.

Reply:
done